app/components/Send/SendPanel/SendRecipientList/SendRecipientListItem/index.jsx

```diff
@@ -48,6 +59,26 @@ class SendRecipientListItem extends Component<Props> {
 
     const { name } = e.target
     let { value } = e.target
+    if (name === 'address') this.setState({ nnsError: '' })
+    if (name === 'address' && value.endsWith('.neo')) {
+      this.setState({ isNnsResolving: true })
```

Comment:
does any of the following code need to happen after setting state? If so, setState takes a callback that we can use. If not, its fine the way it is

Reply:
Nope, this is just to set the loading animation.